test_display_generating_fix.py

```
@@ -0,0 +1,54 @@
+        print(f"❌ ERROR: {agent_file} not found")
+        return False
+    
+    content = agent_file.read_text()
+    
+    # Check that the old problematic patterns are gone
+    old_pattern = r"display_fn=display_generating if \(not stream and self\.verbose\) else None"
+    old_matches = re.findall(old_pattern, content)
+    
+    if old_matches:
+        print(f"❌ FAILED: Found {len(old_matches)} instances of old problematic pattern:")
+        print(f"   'display_fn=display_generating if (not stream and self.verbose) else None'")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)